template/nuxt/nuxt.config.js

```diff
@@ -9,7 +9,6 @@ const { resolve } = require('path')
 <% if (!esm) { -%>
 <% if (ui === 'vuetify') { %>const VuetifyLoaderPlugin = require('vuetify-loader/lib/plugin')<% } %>
```

Comment:
I think it should be removed. Already handled through vuetify module.

Reply:
@robsontenorio
Thank you for the review! It has been deleted!
https://github.com/nuxt/create-nuxt-app/pull/225/commits/2513287084717aeeca136d9663bd175ad2e4967a